src/containers/CircleCI/ApiProvider.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router';
+
+import './css/ApiProvider.css'
+
+class ApiProvider extends Component {
+
+  state = { 
+    input: '',
+    redirect: false, 
+  }
+
+  componentDidMount() {
+    const apiKey = sessionStorage.getItem('CIRCLE_CI_API_KEY');
+    this.setState({redirect: apiKey});
```

Comment:
better use this instead:
  this.setState({redirect: !!apiKey});

This way, you'll always for sure have true/false value in state

Reply:
Ok, tnx for info :)